### PR TITLE
Update database replication documentation to include date type mapping compatibility

### PR DESCRIPTION
**Pull request by pijuskri**

### CODE REVIEW

File: content/en/docs/appstore/modules/database-replication.md [hunk 1]
```diff
@@ -272,13 +272,32 @@ Each import action is executed in a single transaction, which means it is able t
...
+| Mendix type / Java Type | Boolean | util.Date | Integer | Long | Double | Decimal | String |
+| ----------------------- | ------- | --------- | ------- | ---- | ------ | ------- | ------ |
+| Boolean                 | X       |           | X       | X    | X      | X       | X      |
+| DateTime                |         | X         | X       | X    |        |         | X      |
+| Integer                 |         |           | X       | X    | X      | X       | X      |
+| AutoNumber/Long         |         | X         |         | X    | X      | X       | X      |
+| Decimal                 |         |           | X       | X    | X      | X       | X      |
+| HashString/String       |         | X         |         |      |        |         | X      |
+| Enum                    |         | X         |         |      |        |         | X      |
+| Binary                  |         |           |         |      |        |         |        |
```
Comment (dahfjkg) [inline content/en/docs/appstore/modules/database-replication.md]: When importing, we convert from Java type to Mendix type. I'm not sure if my perception is correct, but I would usually expect "from" to be rows and "to" to be columns.

Comment (pijuskri) [reply content/en/docs/appstore/modules/database-replication.md]: Youre right, thats how we normally orientate. Its best to change it then.
https://docs.mendix.com/refguide/attributes-type-migration/

**Comment by MarkvanMents** [comment]
I've converted this to draft for now - please let me know when it is ready for a technical writer's review.

**Comment by dahfjkg** [comment]
Existing content looks good. Based on the recent questions, I would additionally mention that CLOBs are not supported

**Comment by dahfjkg** [comment]
No more comments, so passed technical review

**Comment by pijuskri** [comment]
@MarkvanMents we performed technical review, the MR should be ready for language review.

File: content/en/docs/appstore/use-content/modules/database-replication.md [hunk 2]
```diff
@@ -306,13 +306,37 @@ Each import action is executed in a single transaction, which means it is able t
 
 You can import and export table mappings to an XML file using the **Table mapping** tab.
 
-## 11 Troubleshooting
+## 11 Data types
+ 
+Database-specific data types are converted to a Java type with the JDBC driver. The module then converts these Java types into a Mendix primitive type. This step also performs implicit conversions between types.
 
-### 11.1 A Table Alias Is Selected but the Column Name Field Does Not Show Anything
+[Mendix primitive types](https://apidocs.rnd.mendix.com/9/runtime/com/mendix/systemwideinterfaces/core/meta/IMetaPrimitive.PrimitiveType.html) and supported mappings from java objects are defined below (supported conversions are marked with _X_):
```
Comment (MarkvanMents) [inline content/en/docs/appstore/use-content/modules/database-replication.md]: Would a link to https://docs.mendix.com/refguide/data-types/ be better - to keep the reader on the same site?

File: content/en/docs/appstore/use-content/modules/database-replication.md [hunk 3]
```diff
@@ -306,13 +306,37 @@ Each import action is executed in a single transaction, which means it is able t
 
 You can import and export table mappings to an XML file using the **Table mapping** tab.
 
-## 11 Troubleshooting
+## 11 Data types
+ 
+Database-specific data types are converted to a Java type with the JDBC driver. The module then converts these Java types into a Mendix primitive type. This step also performs implicit conversions between types.
```
Comment (MarkvanMents) [inline content/en/docs/appstore/use-content/modules/database-replication.md:311]: I rewrote this a bit as it wasn't clear. Can you check that it still says what you want it to say. Wasn't sure why you said "attempts" when converting Java types to Mendix types.

**Review by MarkvanMents — COMMENTED** [review]
@pijuskri 
Hi Pijus,
A couple of comments from my proofreading. Let me know what you think.
Other than that, looks ready to publish.

File: content/en/docs/appstore/use-content/modules/database-replication.md (hunk 2 above)
Comment (pijuskri) [reply content/en/docs/appstore/use-content/modules/database-replication.md]: I did initially want to link to this page, but there are differences between the listed types. This module does not support some of the types in `/refguide/data-types/` and directly uses the values shown in the api docs. 

I dont see the data types changing soon however, so I would say its better to just leave it as text without a hyperlink.

File: content/en/docs/appstore/use-content/modules/database-replication.md (hunk 3 above)
Comment (pijuskri) [reply content/en/docs/appstore/use-content/modules/database-replication.md:311]: Meaning looks good to me

**Review by MarkvanMents — APPROVED** [review]
Thanks for your responses @pijuskri 
I'll approve and merge now and it will be published tomorrow,

File: content/en/docs/appstore/use-content/modules/database-replication.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/use-content/modules/database-replication.md]: Thanks for the explanation - I'll leave it as is.

**Pull request merged**